### ISSUE TITLE
CSS output unchanged

### ISSUE THREAD

**Issue by SerbanC** [ilkka/generator-webpack-es6-cssnext#22]
Hello,

I just installed the generator and ran a new app "as-is" by running `webpack` then `npm start`, and the output CSS in `<head>` is exactly the same as the source .css file.

Am I missing something? Shouldn't CSSNext convert it into current day CSS that works on most browsers?

<img width="979" alt="screen shot 2016-04-21 at 12 27 45" src="https://cloud.githubusercontent.com/assets/1150246/14704270/8337cd9e-07bc-11e6-9a8b-2c8e4295ec93.png">

**Comment by SerbanC** [comment]
I managed to get it working on a webpack/postcss/babel setup I made myself from scratch.

In case this will sometime be useful to someone else: I'm using ExtractTextPlugin to get my styles into a separate file, and I had my css loader setup like so:

``` JS
var ExtractTextPlugin = require("extract-text-webpack-plugin");

module.exports = {

(...)

      {
        test: /\.css$/,
        loader: ExtractTextPlugin.extract('style', 'css', 'postcss'),
      },

(...)

  plugins: [
    new ExtractTextPlugin('public/styles.css'),
  ],
}
```

What eventually got it working was changing the `extract` args to (no style-loader required anymore):

``` JS
var ExtractTextPlugin = require("extract-text-webpack-plugin");

module.exports = {

(...)

      {
        test: /\.css$/,
        loader: ExtractTextPlugin.extract('css!postcss'),
      },

(...)

  plugins: [
    new ExtractTextPlugin('public/styles.css'),
  ],
}
```

PS: the packages you're using are deprecated, for example `cssnext` has now switched to `postcss-cssnext`

**Comment by SerbanC** [comment]
I created #23 using what I discovered while playing with my own app setup.

**Comment by josephdburdick** [comment]
This issue has a merged solution, this can be closed.

**Comment by ilkka** [comment]
whoop, should keep tabs on dangling open issues I guess 😅. This is fixed in #23
